mysql-test/suite/galera/t/galera_events3.test

```diff
@@ -15,6 +15,7 @@ call mtr.add_suppression("Operation DROP USER failed");
 call mtr.add_suppression("Event [0-9]* Query apply failed: [0-9]*, seqno [0-9]*");
 CREATE TABLE t1(a INT AUTO_INCREMENT, b INT, PRIMARY KEY pka(a));
 --send SET DEBUG_SYNC = 'now WAIT_FOR has_global_grant.got_lock'; 
+--sleep 1
```

Comment:
Is this sleep needed, will it be stable on jenkins?

Reply:
The sleep is there to make the test more reliable.  It failed (for me) occasionally under load.